### PR TITLE
Annoted `ibm-powervs-tier1` as a default storage class

### PR DESCRIPTION
**Pull request by Madhan-SWE**
Annoted `ibm-powervs-tier1` as a default storage class on creation.

### CODE REVIEW

File: assets/storageclass_tier3.yaml [hunk 1]
```diff
@@ -2,6 +2,8 @@ kind: StorageClass
 apiVersion: storage.k8s.io/v1
 metadata:
   name: tier3-csi
```
Comment (mkumatag) [inline assets/storageclass_tier3.yaml:4]: can we rename this name to something more descriptive e.g: `ibm-powervs-tier3`

File: assets/storageclass_tier3.yaml [hunk 2]
```diff
@@ -2,6 +2,8 @@ kind: StorageClass
...
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
```
Comment (mkumatag) [inline assets/storageclass_tier3.yaml:6]: It will be good if we can hear more from others as well on this option, like `T3` vs `T1` as default

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: assets/storageclass_tier3.yaml (hunk 1 above)
Comment (Madhan-SWE) [reply assets/storageclass_tier3.yaml]: Storage class names changed.

**Review by mkumatag — COMMENTED** [review]
/lgtm
/approve

**Comment by mkumatag** [comment]
/hold

fix the commit message

**Comment by Madhan-SWE** [comment]
Commit message changed.

**Review by mkumatag — APPROVED** [review]
/lgtm
/approve

**Pull request merged**